packages/swirl-components/src/components/swirl-date-input/swirl-date-input.tsx

```diff
@@ -46,6 +46,7 @@ export class SwirlDateInput {
   @Prop() labels?: WCDatepickerLabels;
   @Prop() locale?: string = "en-US";
   @Prop() placeholder?: string = "yyyy-mm-dd";
+  @Prop() preferredInputMode?: "input" | "pick" = "pick";
```

Comment:
Issue: To not make this a breaking change, the default should be "input".

Reply:
Sure, it was just to help me test on mobile :P